include/flexflow/batch_config.h

```diff
@@ -69,6 +70,7 @@ class BatchConfig {
   PerTokenInfo tokensInfo[MAX_NUM_TOKENS];
 
   bool request_completed[MAX_NUM_REQUESTS];
+  bool request_running[MAX_NUM_TOKENS];
```

Comment:
Why do we need `request_running`? What's the difference between this and `request_completed`?

Reply:
It tells the attention kernel to skip attention score computation here: https://github.com/flexflow/FlexFlow/blob/inference/src/ops/spec_inc_multihead_self_attention.cu#L251
This change is in the previous RM pr draft(https://github.com/flexflow/FlexFlow/pull/978/files#diff-9b6c3450973185bf46409fbd3dc6a1810907ba3a0995a40417a46e896b75bbedR251) but has not been made yet in this one.